.github/workflows/pull.yml

```diff
@@ -395,7 +366,7 @@ jobs:
         size=${arr[4]}
         # threshold=48120 on devserver with gcc11.4
         # todo(lfq): update once binary size is below 50kb.
-        threshold="51784"
+        threshold="51504"
```

Comment:
it's awkward that we have the same threshold for two different CPU architectures, but this is still an incremental improvement